pkg/globalconfig/globalconfigtypes/globalconfig_types.go

```diff
@@ -0,0 +1,31 @@
+package globalconfigtypes
+
+type RouterType = string
```

Comment:
```suggestion
type RouterType string
```

We like to have a new type so equal symbol is not needed. With the equal symbol no new type is created.

Reply:
We decided not to do this.